_clients/data-prepper/migrate-open-distro.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: default
+title: Migrating from Open Distro Data Prepper
+parent: Data Prepper
+nav_order: 2
+---
+
+# Migrating from Open Distro Data Prepper
+
+Beginning with Data Prepper 1.1, there is only one distribution of Data Prepper - Open Search Data Prepper. This document helps existing users migrate from the old Open Distro Data Prepper to OpenSearch Data Prepper.
```

Comment:
Could use a colon instead of dash: "... there is only one distribution of Data Prepper: OpenSearch Data Prepper."